### ISSUE TITLE
execute_graphql_request caused warnings

### ISSUE THREAD

**Issue by jedie** [graphql-python/graphene-django#893]
`graphene_django.views.GraphQLView.execute_graphql_request()` will generate these warnings:

```
DeprecationWarning: The 'root' alias has been deprecated. Please use 'root_value' instead.
DeprecationWarning: The 'context' alias has been deprecated. Please use 'context_value' instead.
DeprecationWarning: The 'variables' alias has been deprecated. Please use 'variable_values' instead.
```

The warnings are generated here: `.../graphql/execution/executor.py:92: in execute`

It's graphene-django v2.8.1 and graphene v2.1.8

**Comment by SmileyChris** [comment]
Fixed already in #878, just waiting on a release since then

**Comment by jkimbo** [comment]
This has been fixed. Closing this issue.
